client/src/App.js

```diff
@@ -19,8 +19,10 @@ const App = () => {
   useEffect(() => {
     if (useMock === true) {
       store.dispatch.data.fetchRivers(mockRivers);
+      store.dispatch.editing.fetchRivers(mockRivers);
     } else {
       store.dispatch.data.fetchRiversAsync();
+      store.dispatch.editing.fetchRiversAsync();
```

Comment:
Whereas this effect doesn't exist in `editing` model.

Reply:
Same thing as above, my mistake, taking it out.